Makefile

```diff
@@ -158,6 +158,11 @@ linux-binary: clean bootstrap
 	mkdir -p $(SNAPSHOTDIR)
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -installsuffix cgo -o $(SNAPSHOTDIR)/kai .
 
+.PHONY: linux-binary-fips
+linux-binary-fips: clean bootstrap
+	mkdir -p $(SNAPSHOTDIR)
+	CGO_ENABLED=1 GOOS=linux GOARCH=amd64 GOEXPERIMENT=boringcrypto go build -a -installsuffix cgo -o $(SNAPSHOTDIR)/kai .
```

Comment:
CGO_ENABLED=1 also needs to be set. Not sure how this would impact your build process, i.e it'd need to be compiled against the same distro it's going to run in.

For sure this would need some more of your time to test and think through

Reply:
Thanks for the PR @electricthunder, we're taking a look at our build process to see if we can accommodate this change.